### ISSUE TITLE
MatrixStatsAggregation.php does not comply with psr-4 autoloading standard

### ISSUE THREAD

**Issue by lorenzbausch** [ongr-io/ElasticsearchDSL#319]
When installing this library with Composer the following warning is shown:
```
Deprecation Notice: Class ONGR\ElasticsearchDSL\Aggregation\Matrix\MaxAggregation located in ./vendor/ongr/elasticsearch-dsl/src/Aggregation/Matrix/MatrixStatsAggregation.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
```

**Comment by wawan93** [comment]
#313

**Comment by lanlin** [comment]
```shell
Deprecation Notice: 
Class ONGR\ElasticsearchDSL\Aggregation\Matrix\MaxAggregation located in 
./vendor/ongr/elasticsearch-dsl/src/Aggregation/Matrix/MatrixStatsAggregation.php 
does not comply with psr-4 autoloading standard.
It will not autoload anymore in Composer v2.0. 
in phar:///usr/local/Cellar/composer/1.10.10/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
```

**Comment by 84m** [comment]
Fixed with 8c75b81
